worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java

```diff
@@ -461,13 +464,13 @@ public synchronized <T extends Future<T>> T call(IChunkSet set, Runnable finaliz
                             synchronized (super.sectionLocks[getSectionIndex]) {
                                 LevelChunkSection existingSection = levelChunkSections[getSectionIndex];
                                 if (createCopy && existingSection != null) {
-                                    copy.storeBiomes(getSectionIndex, existingSection.getBiomes().copy());
+                                    copy.storeBiomes(getSectionIndex, existingSection.getBiomes().recreate().copy());
```

Comment:
Recreate does not seem to do the same thing as copy, and instead seems to recreate with no data

Reply:
Did this earlier when IJ refused to index paperweight to make it compile while browsing the jar, but apparently we can't call #copy() on here anymore, same like below?